### PR TITLE
Added one-click reaction documentation

### PR DESCRIPTION
**Pull request by cwarnermm**
Updated documentation for https://mattermost.atlassian.net/browse/MM-26478:
- Channel Basics: added headings, inline images, and one-click reactions
- Using Emoji: added one-click reactions and inline images
- Managing Account Settings: removed unnecessary content duplication in tabs, added one-click user setting

### CODE REVIEW

**Review by wiersgallak — APPROVED** [review]

File: source/messaging/using-emoji.rst [hunk 1]
```diff
@@ -39,6 +42,14 @@ Select the **Skin Tone** icon in the top right corner of the Emoji Picker to spe
 .. image:: ../images/emoji-skin-tone.png
   :alt: Select a default skin tone for people-based emojis.
 
+Selecting recent emojis
+-----------------------
+
+From Mattermost v6.1 and in Mattermost Cloud, react to messages quickly by selecting a recently used emoji while hovering over a message. If your most recent reactions aren't visible, you can enable one-click reactions by selecting **Settings > Display > One-click reactions on messages**.
```
Comment (cwarnermm) [inline source/messaging/using-emoji.rst:48]:
```suggestion
From Mattermost v6.1 and in Mattermost Cloud, react to messages quickly by selecting a recently used emoji while hovering over a message. If your most recent reactions aren't visible, you can enable quick reactions by selecting **Settings > Display > One-click reactions on messages**.
```

Comment (cwarnermm) [inline source/messaging/using-emoji.rst]:
```suggestion
From Mattermost v6.1 and in Mattermost Cloud, react to messages quickly by selecting a recently used emoji while hovering over a message. If your most recent reactions aren't visible, you can enable one-click reactions by selecting **Settings > Display > Quick reactions on messages**.
```

File: source/messaging/using-emoji.rst [hunk 2]
```diff
@@ -39,6 +42,14 @@ Select the **Skin Tone** icon in the top right corner of the Emoji Picker to spe
 .. image:: ../images/emoji-skin-tone.png
   :alt: Select a default skin tone for people-based emojis.
 
+Selecting recent emojis
+-----------------------
+
+From Mattermost v6.1 and in Mattermost Cloud, react to messages quickly by selecting a recently used emoji while hovering over a message. If your most recent reactions aren't visible, you can enable quick reactions by selecting **Settings > Display > One-click reactions on messages**.
```
Comment (cwarnermm) [inline source/messaging/using-emoji.rst:48]:
```suggestion
From Mattermost v6.1 and in Mattermost Cloud, react to messages quickly by selecting a recently used emoji while hovering over a message. If your most recent reactions aren't visible, you can enable quick reactions by selecting **Settings > Display > Quick reactions on messages**.
```

File: source/messaging/managing-account-settings.rst [hunk 3]
```diff
@@ -333,6 +269,11 @@ Collapsed Reply Threads (Beta) offers an enhanced experience for users communica
 
 See our `Organizing Conversations using Collapsed Reply Threads (Beta) <https://docs.mattermost.com/messaging/organizing-conversations.html>`__ documentation to learn more about this feature.
 
+One-click reactions on messages
```
Comment (cwarnermm) [inline source/messaging/managing-account-settings.rst:272]:
```suggestion
Quick reactions on messages
```

File: source/messaging/managing-account-settings.rst [hunk 4]
```diff
@@ -333,6 +269,11 @@ Collapsed Reply Threads (Beta) offers an enhanced experience for users communica
 
 See our `Organizing Conversations using Collapsed Reply Threads (Beta) <https://docs.mattermost.com/messaging/organizing-conversations.html>`__ documentation to learn more about this feature.
 
+Quick reactions on messages
```
Comment (cwarnermm) [inline source/messaging/managing-account-settings.rst:272]:
```suggestion
Quick reactions to messages
```

File: source/messaging/managing-account-settings.rst [hunk 5]
```diff
@@ -333,6 +269,11 @@ Collapsed Reply Threads (Beta) offers an enhanced experience for users communica
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+React to messages with recently used reactions in a single click when hovering over a message.
```
Comment (cwarnermm) [inline source/messaging/managing-account-settings.rst:275]:
```suggestion
You can react to messages quickly using your most recently used emojis by hovering over a message.
```

**Comment by cwarnermm** [comment]
@justinegeffen - We'll be updating the existing "one-click" terminology in-product for this feature to "quick reactions" in a separate PR.

File: source/messaging/channels-basics.rst [hunk 6]
```diff
@@ -18,29 +18,65 @@ Channels Basics
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-**Write messages** using the text input box at the bottom of the screen. Press ENTER to send a message. Use SHIFT+ENTER to create a new line without sending a message. To send messages on CTRL+ENTER and use ENTER to insert new lines go to **Settings > Advanced > Send messages on CTRL/CMD+ENTER**.
+Write messages
+--------------
 
-**Reply to messages** by clicking the reply arrow next to the message text.
+Write messages using the text input box at the bottom of the screen. Press ENTER to send a message. Use SHIFT+ENTER to create a new line without sending a message.
```
Comment (justinegeffen) [inline source/messaging/channels-basics.rst:24]: If the user has set their input to be CMD+Enter to post, should we mention that? In that case they just use ENTER for a new line (and possibly SHIFT+ENTER, I have to test that).

File: source/messaging/managing-account-settings.rst [hunk 7]
```diff
@@ -29,6 +29,9 @@ Account Settings is where you can configure your profile settings, notification
       - Access **Profile** and **Security** settings from your avatar in the Global Header.
```
Comment (justinegeffen) [inline source/messaging/managing-account-settings.rst:29]:
```suggestion
      - Access **Profile** and **Security** settings from your Avatar in the Global Header.
```

File: source/messaging/managing-account-settings.rst [hunk 8]
```diff
@@ -333,6 +269,11 @@ Collapsed Reply Threads (Beta) offers an enhanced experience for users communica
 
 See our `Organizing Conversations using Collapsed Reply Threads (Beta) <https://docs.mattermost.com/messaging/organizing-conversations.html>`__ documentation to learn more about this feature.
 
+Quick reactions to messages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can react to messages quickly using your most recently used emojis by hovering over a message.
```
Comment (justinegeffen) [inline source/messaging/managing-account-settings.rst:275]:
```suggestion
You can react to messages quickly with your most recently-used emojis by hovering over a message.
```

File: source/messaging/managing-account-settings.rst [hunk 9]
```diff
@@ -333,6 +269,11 @@ Collapsed Reply Threads (Beta) offers an enhanced experience for users communica
...
+
 Channel Display
```
Comment (justinegeffen) [inline source/messaging/managing-account-settings.rst:277]:
```suggestion
Channel display
```

File: source/messaging/using-emoji.rst [hunk 10]
```diff
@@ -39,6 +42,14 @@ Select the **Skin Tone** icon in the top right corner of the Emoji Picker to spe
 .. image:: ../images/emoji-skin-tone.png
   :alt: Select a default skin tone for people-based emojis.
 
+Selecting recent emojis
+-----------------------
+
+From Mattermost v6.1 and in Mattermost Cloud, react to messages quickly by selecting a recently used emoji while hovering over a message. If your most recent reactions aren't visible, you can enable quick reactions by selecting **Settings > Display > Quick reactions on messages**.
```
Comment (justinegeffen) [inline source/messaging/using-emoji.rst:48]:
```suggestion
From Mattermost v6.1 and in Mattermost Cloud, react to messages quickly by selecting a recently-used emoji while hovering over a message. If your most recent reactions aren't visible, you can enable quick reactions by selecting **Settings > Display > Quick reactions on messages**.
```

**Review by justinegeffen — COMMENTED** [review]
Some rewording feedback. :)

**Pull request merged**